model/channel.go

```diff
@@ -206,6 +206,10 @@ func (o *Channel) AddProp(key string, value interface{}) {
 	o.Props[key] = value
 }
 
+func (o *Channel) IsGroupConstrained() bool {
```

Comment:
:heart: 

Sometimes I think golang makes me stupid, because I never seem to think of obvious things like this anymore that just make the code so much simpler, more robust and more readable.